src/elvis.app.src

```diff
@@ -11,6 +11,33 @@
       , elvis_webhook
       ]
     }
+  , { env
+    , [ { config
+        , [ #{ dirs => [ "apps/*/src"
```

Comment:
I know this PR is closed already, but I think we can improve on it still (I don't mind creating a new PR, but would like input before I do):
1. how about using `apps/**` instead of a specific list of sub-folders? (also applies to `makefiles`, further down),
2. the same goes for `src/**` and `test/**` (also applies to `makefiles`, further down),

[I don't know where `lib/**` comes from :(]

Reply:
I like your ideas. Go ahead! 👍